php-cas/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN yum -y update \
     && yum -y install httpd mod_ssl php php-pear php-xml php-pdo wget \
     && yum -y clean all
 
-RUN pear install http://downloads.jasig.org/cas-clients/php/current.tgz
+RUN pear install https://developer.jasig.org/cas-clients/php/current.tgz
```

Comment:
Yesterday, the URL didn't work for me so it was impossible to `docker-compose build`.
Thanks for the quick merge!